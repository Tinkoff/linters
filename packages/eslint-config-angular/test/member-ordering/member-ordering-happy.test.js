import ESlint from 'eslint';
import path from 'path';

describe('member-ordering / happy path', () => {
  const cli = new ESlint.CLIEngine({
    cwd: path.join(__dirname, '..'),
    useEslintrc: false,
    baseConfig: {
      extends: ['../ts/member-ordering'],
    },
  });

  it('happy', () => {
    const codeframe = cli.getFormatter('codeframe');
    const report = cli.executeOnFiles([
      path.join(__dirname, './__fixtures__/member-ordering-happy.fixture.ts'),
    ]);

    expect(codeframe(report.results)).toMatchSnapshot();
  });
});
