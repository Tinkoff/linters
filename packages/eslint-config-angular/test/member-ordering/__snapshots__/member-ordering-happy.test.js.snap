// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`member-ordering / happy path happy 1`] = `""`;
